### ISSUE TITLE
Versionize linuxdeployqt

### ISSUE THREAD

**Issue by Silex** [probonopd/linuxdeployqt#282]
Hello,

Recently in #35 we saw that there was somewhat of a regression, which brings us to the versioning question.

Why not do releases? I know this is still somewhat "beta" software but having versions would allow us to use a specific working version instead of "continuous" which can break from time to time.

**Comment by probonopd** [comment]
Yes, you are right. Reasons are: Things are still "moving", and releasing takes time and effort. Most likely I'd do time-based (automated) releases if I find a script that automate it.

**Comment by Silex** [comment]
Yeah time-based releases would be ok.

**Comment by knapsu** [comment]
I encountered problems with continuous few times breaking my builds. I use it because there are only binaries for `continuous` on GitHub releases page. It would be good to have other not changing releases to make depending project builds more reproducible.

**Comment by probonopd** [comment]
Agree... anyone volunteers to be the Release Manager?

**Comment by goolhanrry** [comment]
> Agree... anyone volunteers to be the Release Manager?

Sorry but what should a Release Manager specifically do for the repository? I'd like to be a volunteer : )

**Comment by Silex** [comment]
@goolhanrry: a release manager tag versions and usually keeps a changelog up to date.
